EXT_BeamPhysics.md

```diff
@@ -0,0 +1,269 @@
+Extension to the openPMD Standard for Describing Particle Beams and X-rays
+==========================================
+
+Version 1.0.0 (March 1, 2018)
+
+Overview
+------------
+
+The `BeamPhysics` extension to the openPMD standard is meant for describing particles and fields commonly encountered
+in accelerator physics simulations.
+
+How to Use this Extension
+---------------
+
+The `BeamPhysics` extension to the openPMD standard is indicated in a data file by the setting of the `openPMDextension` attribute:
+```
+  openPMDextension = BeamPhysics, SpeciesType
+```
+
+Note: The `SpeciesType` extension must be used when using the `BeamPhysics` extension.
+
+Definitions
+-----------
+
+- **Lattice**: A **lattice** is the arrangement of elements in a machine such as a particle accelerator.
+
+- **Global coordinate system**: The **global** coordinate system is a coordinate system that is
+used to describe the position and orientation of machine elements in space. That is, the **global**
+coordinate system is fixed with respect to the building or room where the machine is placed independent of the
+machine itself.
+
+- **Lattice coordinate system**: The curvilinear coordinate system whose "longitudinal"
+coordinate (typically called **s**) typically runs through the nominal centers of the elements
+in the machine. Typically, the **lattice** coordinate system is used to describe misalignments
+of lattice elements with respect to the rest of the lattice.
+
+- **Group**: A **group** is a container structure containing
+a set of zero or more **attributes**, a set of zero or more **groups** (which can be called
+**sub-groups**), and a set of zero or more **datasets**. Note: In HDF5, these are also
+called **groups**.
+
+- **Dataset**: A **dataset** is a structure that contains a set of zero or more **attributes** and a
+data aray (which may be multidimensional).  Note: In HDF5, these are also called **datasets**.
+
+- **Record**: A **record** is a **group** (without any sub-groups) or a **dataset** that contains data
```

Comment:
@RemiLehe 

I agree that definitions should only appear once so this should be revised. "Data set" is not defined in the base Standard so I would propose just moving the definition here to the base Standard. [By the way, I use "dataset" rather than "data set". I don't care what the convention is but we should use one or the other but not both.]

Also "record" in the base Standard is used before it's definition. I think it would be good to have a definitions section in the base Standard near the top of the file so that words are defined before they are used.

Reply:
OK, I think that these definitions could be included in the base standard. 
Since this is a bit of a different topic, I suggest removing them from the current pull request, and opening another pull request where these definitions are added in the base standard. Would that be okay with you?

Reply:
OK I will work on it...